polaris-core/src/main/java/org/apache/polaris/core/connection/ConnectionType.java

```diff
@@ -32,6 +32,7 @@
 public enum ConnectionType {
   NULL_TYPE(0),
   ICEBERG_REST(1),
+  HADOOP(2),
```

Comment:
Do we plan to federate to `Hadoop` through a non-Iceberg `Catalog`?
Just curious how generic tables would work with Catalog Federation in that case.

Reply:
This is for Hadoop's `IcebergCatalog`, but good point. For UC, we might have a separate connection for UC's IRC endpoints compared to the non-IRC ones and they might be used for different table types